test/stage2/sparcv9.zig

```diff
@@ -0,0 +1,39 @@
+const std = @import("std");
+const TestContext = @import("../../src/test.zig").TestContext;
+
+const linux_sparcv9 = std.zig.CrossTarget{
+    .cpu_arch = .sparcv9,
+    .os_tag = .linux,
+};
+
+pub fn addCases(ctx: *TestContext) !void {
```

Comment:
You need to call this function in https://github.com/ziglang/zig/blob/2a00df9c091498268b58dd671f646a5590439b7a/test/cases.zig#L9-L21 for the tests to be actually executed.

Reply:
Ah, added, thanks.